src/js/populationSlider.ts

```diff
@@ -69,6 +68,7 @@ const setUpPopulationSlider = (
   searchElement: Choices,
   sliders: PopulationSliders
 ): void => {
+  draw(sliders, 0, RANGE_MAX);
```

Comment:
Moved `draw` into `setUpPopulationSlider` since this will be called once when user opens filter popup.